src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -76,6 +76,7 @@
     <RequiresSystemIOAbstractions>true</RequiresSystemIOAbstractions>
     <RequiresLibGit>true</RequiresLibGit>
     <RequiresChannels>true</RequiresChannels>
+    <RequiresLanguageServices>true</RequiresLanguageServices>
```

Comment:
Added a new MSBuild property so projects can indicate that they need the package.